libraries/standard/mqtt/integration-test/CMakeLists.txt

```diff
@@ -58,12 +58,34 @@ create_test(${stest_name}
             "${test_include_directories}"
         )
 
-# Copy the certificates and client key to the binary directory.
-add_custom_command(
-    TARGET
-        ${stest_name}
-    POST_BUILD
-        COMMAND ${CMAKE_COMMAND} -E copy_directory
-        "${CMAKE_CURRENT_LIST_DIR}/certificates"
-        "$<TARGET_FILE_DIR:${stest_name}>/certificates"
-)
+# Set preprocessor defines for test if configured in build.
+if(BROKER_ENDPOINT)
+    target_compile_definitions(
+        ${stest_name} PRIVATE
+            BROKER_ENDPOINT="${BROKER_ENDPOINT}"
+    )
+endif()
+if(ROOT_CA_CERT_PATH)
+    target_compile_definitions(
+        ${stest_name} PRIVATE
+            SERVER_ROOT_CA_CERT_PATH="${ROOT_CA_CERT_PATH}"
```

Comment:
Should this be an absolute path?

Reply:
It is supposed to be provided by the user through CMake configuration.

Reply:
Something of this sort is needed? https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/demos/CMakeLists.txt#L58

Reply:
I was referring to the CMake configuration command passed by the user. 
I don't see much value in pre-pending the absolute path with $PWD as that would pre-pend the path (to the certificate) from where the cmake command is run (which is most likely in the `build` directory), and can cause unexpected failure to occur due to the path pre-pending. As the CI team will be providing the absolute path to the credential files in their CMake command, I think that suffices.